index.html

```diff
@@ -7,10 +7,43 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio: Ahmed Hasan Rony </title>
     <link rel="stylesheet" href="css/style.css">
+    <!-- fontawesome cdn -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+        integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 
-<body>
-    <h1 class="hello">Portfolio</h1>
+<body class="portfolio">
+    <header class="header">
+        <nav class="portfolio__navbar">
+            <div class="navbar__logo">
+                <a href=#>Welcome</a>
+            </div>
+            <div class="navbar-button">
+                <a href="#"><img src="images/button.png" alt="nav-button" height="20px"></a>
+            </div>
+        </nav>
+    </header>
+    <div class="main__content">
+        <div class="main__heading">
+            <h1>Hey There. I'm Rony<br><span>I'm a Software Developer</span></h1>
+        </div>
+        <div class="main__description">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="social__icons">
+            <a href="#"><img src="images/twitter.svg" alt="twitter" height="20px"></a>
+            <a href="#"><img src="images/Linkedin icon.svg" alt="twitter" height="20px"></a>
+            <a href="#"><img src="images/medium.svg" alt="twitter" height="20px"></a>
+            <a href="#"><img src="images/github.svg" alt="twitter" height="20px"></a>
+            <a href="#"><img src="images/angellist.svg" alt="twitter" height="20px"></a>
+        </div>
+        <div class="action__button">
+            <a href="#"><img src="images/Action.svg" alt="action__button"></a>
+        </div>
```

Comment:
- [ ] For convention, and semantics reasons, I will adivse you use ul tag and have the a tags inside the li tag each.

Reply:
okay i will put these in ul tag